.circleci/config.yml

```diff
@@ -64,7 +64,14 @@ jobs:
           command: go test -cover -tags integration -v ./...
       - run:
           name: Build binary
-          command: go build -o ./build/_output/bin/dynatrace-oneagent-operator ./cmd/manager
+          command: |
+            if [[ -z "$CIRCLE_TAG" ]]; then
+               version="snapshot-${CIRCLE_BRANCH}"
+            else
+               version="${CIRCLE_TAG}"
+            fi
+            echo ${version}
+            go build -ldflags="-X \'github.com/Dynatrace/dynatrace-oneagent-operator/version.Version=${version}\'" -o ./build/_output/bin/dynatrace-oneagent-operator ./cmd/manager
```

Comment:
Can you remove the slashes for the single quotes? It's making it not set the correct field.

```
$ docker run quay.io/dynatrace/dynatrace-oneagent-operator:snapshot-APM-207673-add-tag-per-branch dynatrace-oneagent-operator 2>&1 | grep "Version of dynatrace-oneagent-operator"
{"level":"info","ts":"2019-12-04T12:53:00.695Z","logger":"cmd","msg":"Version of dynatrace-oneagent-operator: snapshot"}
```

Reply:
done